src/App.js

```diff
@@ -1,15 +1,21 @@
+
 import React from 'react';
-import TodoList from './TodoList'; // Importing the TodoList component
-import AddTodoForm from './AddTodoForm'; // Importing the AddTodoForm component
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 
 function App() {
-  return (
-    <div>
-      <h1>ToDo List</h1>
-      <AddTodoForm /> {/* Using the AddTodoForm component*/}
-      <TodoList /> {/* Using the TodoList component*/}
-    </div>
-  );
+    const [newTodo, setNewTodo] = React.useState("");
```

Comment:
Nice job accessing `useState` and providing a default value. Although you won't always get an error for not providing a default value, it's a good habit to get into. It'll be helpful in the long run to provide a default value.